cpp/core/utils/Print.h

```diff
@@ -1,298 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-#pragma once
-
-#include <iostream>
-#include <string>
-#include <vector>
-#include "DebugOut.h"
-
-namespace gluten {
-
-// the functions defined in this file are used to debug
-// the token `ToString` means the method of `ToString()`
-// the token `2String` means the method of `toString()`
-
-#ifdef GLUTEN_PRINT_DEBUG
-
-template <typename T>
-static inline void print(const T& t) {
-  std::cout << t;
-}
-
-template <typename T>
-static inline void printLf(const T& t) {
-  std::cout << t << std::endl;
-}
-
-template <typename A, typename B>
-static inline void print(const A& a, const B& b) {
-  std::cout << a << b;
-}
-
-template <typename A, typename B>
-static inline void printLf(const A& a, const B& b) {
-  std::cout << a << b << std::endl;
-}
-
-template <typename A, typename B>
-static inline void printSplit(const A& a, const B& b, const std::string split = ": ") {
-  std::cout << a << split << b;
-}
-
-template <typename A, typename B>
-static inline void printSplitLf(const A& a, const B& b, const std::string split = ": ") {
-  std::cout << a << split << b << std::endl;
-}
-
-template <typename A, typename B>
-static inline void printEq(const A& a, const B& b) {
-  std::cout << a << " = " << b;
-}
-
-template <typename A, typename B>
-static inline void printEqlf(const A& a, const B& b) {
-  std::cout << a << " = " << b << std::endl;
-}
```

Comment:
So there is no code relying on these APIs as of now?

cc @zuochunwei

Reply:
just wait CI.